### DISCUSSION TITLE
nuclei扫着扫着就出现很多乱码

### DISCUSSION THREAD

**Discussion by web1919** [projectdiscovery/nuclei#5422] [General]
为啥nuclei扫完之后会出现大量的请求，扫完的数据也看不到

**Comment by ehsandeep** [comment]
@web1919 do you have example to share? also what command / OS you are using?

**Reply by web1919** [reply]
nuclei -u  后面更的目标  然后扫着就崩溃了

**Reply by ehsandeep** [reply]
@web1919 do include screenshots / example to understand what you are talking about.

**Reply by web1919** [reply]
![image](https://github.com/user-attachments/assets/1ef4d872-0fdb-4483-a75b-8b76d9272134)

**Reply by web1919** [reply]
现在是这个报错

**Reply by ehsandeep** [reply]
@web1919 this is not an error, it's a feature to skip / avoid scanning target that are not responding, you can customize this as needed, also make sure to update to latest version.

```
   -mhe, -max-host-error int  max errors for a host before skipping from scan (default 30)
```

**Reply by 000b000** [reply]
想要看实时响应结果，可以添加-debug 参数。还有，在英文项目里是不是用英文沟通会更方便呢？

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
